src/lib/directive.ts

```diff
@@ -64,3 +64,17 @@ export const langChanged = directive((cb: LangChangedDirectiveCallback) => (part
  * @param config
  */
 export const translate = (key: string, values?: Values | ValuesCallback, config?: ITranslateConfig) => langChanged(() => get(key, values, config));
+
+/**
+ * A lit directive that updates the translation (including rendering html from the translation string) when the language
+ * changes.
+ * @param key
+ * @param values
+ * @param config
+ */
+export const translateUnsafeHTML = (key, values?, config?) => langChanged(() => {
+	const tl = get(key, values, config);
+	const template = document.createElement('template');
+	template.innerHTML = tl;
+	return document.importNode(template.content, true);
+});
```

Comment:
Looks good 👍  I have two suggestions for the implementation.

1. We use the `unsafeHTML` function from `lit-html/directives/unsafe-html` to get the benefits of the [`lit-html` implementation](https://github.com/Polymer/lit-html/blob/master/src/directives/unsafe-html.ts).

2. We change the name of the function to `unsafeTranslate` to stay consistent with the naming of the `unsafeHTML` function.

The suggestions are shown in the code below.

```js
export const unsafeTranslate = (key: string, values?: Values | ValuesCallback, config?: ITranslateConfig) => langChanged(() => unsafeHTML(get(key, values, config)));
```